plasma_framework/contracts/src/vaults/Erc20Vault.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.5.0;
+
+import "./Vault.sol";
+import {PaymentTransactionModel as DepositTx} from "../transactions/PaymentTransactionModel.sol";
+import {IERC20 as IERC20} from "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
```

Comment:
I was following these guidelines:
```
This form is not recommended for use, because it unpredictably pollutes the namespace. 
```

from here https://solidity.readthedocs.io/en/v0.5.9/layout-of-source-files.html#importing-other-source-files

but I actually agree with you and prefer the simpler form of
```
import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
```

Reply:
ah....did not know about that.

How about this one then? We can start changing our convention on solidity import further on.
```
import "filename" as symbolName;
```

Reply:
But then you have to use  `symbolName.symbolName` in the code, which is kind of ugly. 
E.g. if you do:
```
import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol" as IERC20;
```
then `IERC20` refers to the file scope and you need to use `IERC20.IERC20` to refer to the exported contract.

To be honest, I would just stick with using 
```
import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
```

I understand _why_ it's not recommended, but I think the vast majority of contracts are just going to export one global object - the contract itself.

Reply:
oh... :( again wasn't aware we need to `symbolName.symbolName`....
Okay let's stick with our existing version then XDD : )